jack-matchmaker.conf

```diff
@@ -0,0 +1,11 @@
+#
+# Parameters to be passed to jack-matchmaker
+#
+
+PATTERN_FILE="/etc/jack-matchmaker/patterns.txt"
+PATTERNS=""
+CLIENT_NAME="jack-matchmaker"
+CONNECT_INTERVAL=3
```

Comment:
This is missing in the command line (option `-I`).

Reply:
Added.